### PR TITLE
apple-clone webpage. 

### PR DESCRIPTION
**Pull request by Jmagero**
This is a clone of apple web page
-Fixed lint errors
- width of the page should covers the entire window
-applied two colors to navigabation bar

### CODE REVIEW

**Comment by majovanilla** [comment]
# Code Review in progress 👀 by Maria Reyes

File: README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
-# Project Name
+# Apple clone webpage
 
-> One paragraph statement about the project.
+> This project is a clone of apple webpage.
```
Comment (majovanilla) [inline README.md]: - [ ] Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...

- [ ] Delete the section that you are not using.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>apple-clone</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+</head>
+<body>
+    <main>
```
Comment (majovanilla) [inline index.html]: - [ ] Your project is not behaving like the original page. It should be centered on the screen (including the `aside` and the `footer`. You can **start** giving width to your body and then using `margin: auto` to center it and then arranging the rest.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                    <i class="fa fa-play-circle-o" aria-hidden="true"></i>
+                </p>
+            </div>
+        </div>
+        <aside>
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s_2x.jpg"  alt="iPhone 5s. Forward thinking.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c_2x.jpg" alt="iPhone 5c. For the colorful.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years_2x.jpg"  alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+        </aside>
```
Comment (majovanilla) [inline index.html]: - [ ] This section is aligned to the left side of the page, but it should be centered and aligned with the main background image.
Screenshot from 2020-03-16 17-20-26

You might not be able to see this due to your smaller screen, that's why the importance of giving the body a width (less than 100%)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                        <button>iTunes</button>
+                    </li>
+                    <li>
+                        <button>Support</button>
+                    </li>
+                    <li class="right button">
+                        <i class="fa fa-search custom-user"  aria-hidden="true"></i><input type="search" name="search">
+                    </li>
+                </ul>
+            </nav>
```
Comment (majovanilla) [inline index.html]: - [ ]  Read this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) and this [question](https://stackoverflow.com/questions/21715162/when-should-i-use-a-button-button-or-a-link-a-in-html) about the use of buttons. You shouldn't use them the way you are using them right now (please apply the above advice to change them).

- [ ]  The `nav` tag shouldn't be inside the `main` tag. It should be before.

File: index.html (hunk 3 above)
Comment (majovanilla) [inline index.html]: - [ ] The elements should be links like the original page.

File: styles/style.css [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+  border-radius: 0 20px 20px 0;
+  height: 20px;
+  padding: 0;
+  width: 60px;
+}
+
+aside {
+  display: flex;
+  padding: 2px;
+}
```
Comment (majovanilla) [inline styles/style.css]: - [ ] you don't need the padding here. It creates a space above the aside section that shouldn't be there.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+                    </li>
+                </ul>
+            </div>
+            <div class ="second-footer">
+                <span>
+                    Copyright © 2014 Apple Inc. All rights reserved. 
+                    <a href="www.apple.com">Terms of Use</a>
+                    <a href="www.apple.com">Privacy Policy</a>   
+                </span>
+            </div>
```
Comment (majovanilla) [inline index.html]: - [ ] This second footer should be aligned to the left with the `first-footer` and it's not. See image:
Screenshot from 2020-03-16 17-41-43

File: index.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+            <div>
+                <h1>What will your verse be?</h1>
+                <p class="sub-heading">Everyone has something to share.See how people are using
+                    <br>iPad in their unique ways to add to the world's story
+                    <br>
+                    Watch the "Your verse" film and explore the stories
+                    <i class="fa fa-play-circle-o" aria-hidden="true"></i>
+                </p>
+            </div>
+        </div>
```
Comment (majovanilla) [inline index.html]: - [ ] These elements are links. If you check the original page you can see there is no need to write the text, you can just copy the images and position them. Choose an option, but make sure they are clickable and that the look is the same (they should have more space between them)

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey @Jmagero , 👋

## Good job on this project! 👍

- Good Readme file

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

-Please write meaningful pull request messages. Check this [example](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) And ALWAYS include a message in your PR.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔
- Your project would look sooo much better if you include the hover effects on the links of the page, like the original version.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Tolulope Olusakin(TSE)
Please check on this in a few minutes for some feedback! :+1:

**Comment by nikasulo** [comment]
# Great work! @Jmagero :clap: 
Please consider my comments below, make the necessary changes and request a re-review! I wish you the best and see you at the very top :rocket: 

#### Full-width site
Please consider that the original site is full-width, stretching from one end of the browser window to the next. Please consider this and try to replicate it in your project, as right now you have some space between the browser edge and the content of the page

#### Stickler errors
You have done very nicely by including stickler in your project as requested! However, you need to ensure that all errors it throws are appropriately addressed before you request a PR review. Please look into this, address appropriately and request another review. 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi @Jmagero, congratulations on working on the comments of the previous reviewers 👍 We are almost there 👍 We left out some of the comments by the previous TSE. Let us fix the comments below:

* The width of the page should cover the entire window as it is in the original. Check images for reference. Currently
fw1
```Original```
fw
* Let us work on the nav a little bit. Let us apply 2 shades of color to the nav as it is in the original and center the nav

Kindly apply these changes and submit another code review 😄 
_Happy coding_ 👨‍💻 
Find me on slack Daniel Larbi Addo

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo

**Review by addod19 — APPROVED** [review]
## Status: Approved 🏆 🎈 

Hi @Jmagero, **Congratulations** on the great job 👌 You did 👍  implement all the changes requested by the previous TSE'S. Our project now meets all the minimum requirements. I hereby approve of the project. Go ahead and merge to master and submit project completion form 👍 

Good luck with your next projects ⚙️ 
_Happy coding_ 👩‍💻 
Daniel Larbi Addo

**Comment by Jmagero** [comment]
Thanks Daniel

**Pull request merged**